CHANGELOG.md

```diff
@@ -1,69 +1,103 @@
-# PWA Studio Release 12.4.0
+# PWA Studio Release 12.5.0
 
-**NOTE:**  
-_This changelog only contains release notes for PWA Studio and Venia 12.4.0_  
+**NOTE:**
+_This changelog only contains release notes for PWA Studio and Venia 12.5.0_.
 _For older release notes, see_ [PWA Studio releases][].
 
-## Highlights  
+## Highlights
 
-The main focus of PWA Studio 12.4 is additional support for working with both system attributes and custom product attributes. While previously, this could be done, results were difficult to parse, with little ability for sorting and filtering. Now, system and custom product attributes are returned in their own object within the GraphQL response, which can be sorted and filtered.
+The 12.5.0 release of PWA Studio focuses on 2 areas:
 
-### Product attributes
+- A new eventing framework
+- Accessibility improvements
 
-- Products can now be sorted by the product attribute value on the product listing page. — [3761][]
-- The Venia sample data has been updated to include product attributes. — [23][] 
-- Product attribute values can now be used to search for products. — [3747][]
+### Eventing
 
-### Product Attributes on PDP
+We have added a new eventing framework into PWA Studio. This framework allows users to send user event data to Adobe's Edge collection service. The framework includes a [series of events](https://developer.adobe.com/commerce/pwa-studio/integrations/analytics/event-reeference/) by default. Developers can extend the framework by [writing your own events](https://developer.adobe.com/commerce/pwa-studio/integrations/analytics/custom-events/) and you can also configure the framework so that 3rd party modules can subscribe to these events.
```

Comment:
We should avoid associating the eventing framework with Adobe's analytics service or at least be more clear on what it is we are delivering in this release to prevent confusion.

The framework itself is service agnostic. All it does out of the box is allow extensions to subscribe to events being published in the storefront application. What happens after that is up to the extension, e.g. the sample extension just logs that event while the experience platform connector extension sends those events to Adobe's analytics service.

Here's a proposed summary for this section:

> The main feature introduced in PWA Studio 12.5 is the eventing framework.
>
>This framework lets developers write code that subscribes to event data published by the application.
>This data can be sent to different analytics services, such as the Adobe Experience Platform.
>
>The framework also provides an API that lets extension developers send out events in their own components.
>A sample extension is included in this release to serve as a starting point or tutorial for developers.
>
>This release also include the Experience Platform Connector, a PWA Studio extension that can be installed into a storefront project.
>This extension leverages the eventing framework to collect and send data to the Adobe Experience Platform.